.github/workflows/main.yml

```diff
@@ -10,14 +10,14 @@ on:
   push:
     branches:
       - master
-      - [0-9].[0-9].x
+      - "[0-9]+.[0-9]+.x"
     tags:
       - "*"
 
   pull_request:
     branches:
       - master
-      - [0-9].[0-9].x
+      - "[0-9]+.[0-9]+.x"
 
 jobs:
   build:
```

Comment:
Fixes "You have an error in your yaml syntax on line 12" (https://github.com/pytest-dev/pytest/runs/414174858).

Reply:
It appears to handle it as regexp, according to https://github.com/pytest-dev/pytest/commit/dc5188e1cd72de87002b986a9163beb6be5f28a9 where a build was not triggered, but with https://github.com/pytest-dev/pytest/commit/26b5b037e0bd6c459cc73169536de28646818153.

Reply:
Squashed and cherry-picked into https://github.com/pytest-dev/pytest/commit/ce033beb97b91020c1ea8df14c30c43e3477feb0.